.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -160,7 +160,7 @@ jobs:
           max_attempts: 2
           substrings_indicating_flaky_execution: ${{ matrix.suite.flaky_indicators || '' }}
           command: >
-            VIDEO_RECORDING_IS_ENABLED=0;
+            VIDEO_RECORDING_IS_ENABLED=1
```

Comment:
Do you still want this in the final PR?

Also just to check a couple of things:
- Did video recording actually work? I don't see any videos in the artifacts.
- Is the semicolon supposed to be there or not? If not then let's not reintroduce it (when you're changing this back to 0).

Reply:
The video recording worked for e2e tests. The semicolon is not supposed to be there. Reverting the value to 0 without the semicolon